samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AssessmentSettingsMessages.properties

```
@@ -377,7 +377,7 @@ extendedTime_entry_table_title=Add/edit an extended time entry.
 # Extended Time Errors
 extended_time_user_and_group_set=A user or group must be selected to add a date and/or time limit exception.
 extended_time_due_earlier_than_available=The Due Date cannot be earlier than the Available Date for the exception set for {0}.
-extended_time_due_in_past=The Due Date cannot be in the past for the exception set for {0}.
+extended_time_due_in_past=The Due Date is in the past for the exception set for {0}.
```

This is a good start. I would expand on this text, something like:

```suggestion
extended_time_due_in_past=The Due Date is in the past for the exception set for {0}. Please change the Due Date, or Save again to proceed as is.
```

@fostersdesign, any suggestions here?

---

Changed to your suggestion.